addOns/bruteforce/src/main/java/org/zaproxy/zap/extension/bruteforce/OptionsBruteForcePanel.java

```diff
@@ -436,9 +437,7 @@ public void saveParam(Object obj) throws Exception {
     private JSlider getSliderThreadsPerScan() {
         if (sliderThreadsPerScan == null) {
             sliderThreadsPerScan =
-                    new PositiveValuesSlider(
-                            BruteForceParam.DEFAULT_THREAD_PER_SCAN,
-                            BruteForceParam.MAXIMUM_THREADS_PER_SCAN);
+                    new PositiveValuesSlider(Constants.getDefaultThreadCount(), Integer.MAX_VALUE);
```

Comment:
Needs to be replaced with a number spinner per core comment about the performance issue (some of the other add-ons need to be tweaked as well).